src/diagnostics/Directory.Build.props

```diff
@@ -18,7 +18,7 @@
     <RepositoryType>git</RepositoryType>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <EnableSourceControlManagerQueries>true</EnableSourceControlManagerQueries>
-    <GitHubRepositoryName>diagnostics</GitHubRepositoryName>
+    <RepositoryName>diagnostics</RepositoryName>
```

Comment:
do we still need this?

Reply:
Yes. diagnostics conditions on it in a project file that exists in both runtime and diagnostics.

Reply:
ah so RepositoryName is not initialized at all normally?

Reply:
Correct. RepositoryName is only defined inside the VMR. GitHubRepositoryName was never defined aside from in DotNetBuild.props / SourceBuild.props files or in those few cases in a repo.

Reply:
ok, makes sense